### PR TITLE
Fix appliance build issues

### LINKED ISSUE #214: Appliance Build documentation issues
**Describe the bug**
The build documentation for the appliance located at https://vmweventbroker.io/kb/contribute-appliance is not accurate or detailed enough to successfully build the VEBA Appliance.

**To Reproduce**
I believe the following issues exist:
1. `jq` must be installed 
2. An ESXi host must be specified - vCenter doesn't work
3. The host must have SSH enabled
4. The host must have GuestIPHack enabled via `esxcli system settings advanced set -o /Net/GuestIPHack -i 1`
5. VNC port range needs to be opened on the ESXi host
6. Build host must have firewall disabled to allow packer http server to be accessed
7. The build host must be on the same network as the packer target VM, which must be connected to a standard port group on a standard vSwitch. 
8. Command to run is `./build.sh master`
9. There is a bug in `build.sh` where it fails to detect the `jq` binary if not installed in `/usr/local/bin`. This affects CentOS and possibly others.
10. Builds against ESXi 7+ don't work unless using packer > 1.6.3 and the `vnc_over_websocket` option.

**Expected behaviour**
Appliance build should work correctly by following the documented procedure. 

**Screenshots**

**Version (please complete the following information):**
 - VEBA Form Factor: source code
 - VEBA Version: v0.4.1

**Additional context**

### PR DESCRIPTION
**Pull request by dsbibby**
Added support for Packer 1.6.x to support ESXi 7.0+
fixed bug in build.sh on CentOS
added min packer version
Updates to docs

Signed-off-by: David Bibby <davidsbibby@gmail.com>

## Summary

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

* Closes #214 

## Testing Verification

* Build now works on CentOS development machine
* Tested using Packer 1.6.4 with ESXi 6.7 and ESXi 7.0
* Built ova deployed successfully

## Additional Information

* This change makes the minimum Packer version required to be 1.6.3
* This change makes the minimum ESXi version required to be ESXi 6.7 for the build host

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

File: manual/add_ovf_properties.sh [hunk 1]
```diff
@@ -3,25 +3,33 @@
 # SPDX-License-Identifier: BSD-2
 
 OUTPUT_PATH="../output-vmware-iso"
+OVF_PATH=`find ${OUTPUT_PATH} -type f -iname ${VEBA_APPLIANCE_NAME}.ovf -exec dirname "{}" \;`
```
Comment (lamw) [inline manual/add_ovf_properties.sh:6]: Nit: Could we use `$()` notation foe ease of reading to encapsulate the command rather than ```?

**Review by lamw — CHANGES_REQUESTED** [review]
Minor change, but this looks good and thanks for contribution David!

File: manual/add_ovf_properties.sh (hunk 1 above)
Comment (dsbibby) [reply manual/add_ovf_properties.sh]: Yep! Have fixed that and updated.

**Comment by dsbibby** [comment]
Suggested change applied and re-tested... all looks good.
Let me know if there is anything more or if this needs a squash.

**Comment by lamw** [comment]
LGTM, please squash and I'll merge shortly

**Comment by dsbibby** [comment]
All done, thank you!

**Review by lamw — APPROVED** [review]
LGTM

**Pull request merged**